index.html

```diff
@@ -7,6 +7,25 @@
     <link rel="stylesheet" href="index.css">
 </head>
 <body>
-    
+    <div class="wrapper">
+        <div class="main">
+            <div class="row1">
+                <div class="row1_box1 box">BOX1</div>
```

Comment:
remove number form classname
use appropriate class name.
use row_first or row_second like this.